fastlane/lanes/localization.rb

```diff
@@ -169,9 +169,6 @@
     metadata_dir = File.join('fastlane', APP_SPECIFIC_VALUES[app.to_sym][:metadata_dir], 'android')
     version_code = android_get_release_version['code']
 
-    # TODO: This is currently a separate lane so we can invoke it separately, but if after a couple of sprints we see that it works well,
-    #       it we would make sense to remove the lane and instead directly use `skip_upload_metadata: (options[:track] != 'production')`
-    #       in the `upload_build_to_play_store` lane? (especially if we do the same for `skip_upload_changelogs` and get rid of `android_create_xml_release_notes`)
```

Comment:
This `skip_upload_metadata: (options[:track] != 'production'` idea has already been applied and taken care of by https://github.com/wordpress-mobile/WordPress-Android/pull/16235/files#r840866181 which is why this TODO is ok to remove now.

I've decided to still keep that lane around, even if the automation of the upload of both the metadata (app title, description…) and changelogs (aka release notes) have been automated during `upload_build_to_play_store` by https://github.com/wordpress-mobile/WordPress-Android/pull/16235/files#r840866181, because I figured it could still be useful to trigger an update of the metadata manually / on demand and decoupled from the upload of a new build. Especially since Google allows for the metadata (Play Store Listing) to be updated separately from submitting a new version of the app if we want to.